### DISCUSSION TITLE
Specify Timezone in Date

### DISCUSSION THREAD

**Discussion by RickCogley** [retypeapp/retype#215] [Ideas]
Hi - shouldn't you be able to specify the timezone in `date` in the frontmatter? This is typical in my experience:

```
2021-11-21T16:55:13+09:00
```

Since you can use Github Actions' cron feature to schedule builds, future posts should be possible, and in that case, the timezone should be exact.

**Comment by geoffreymcgill** [comment]
Dealing with timezone offsets hurt my brain. We will look into this.

**Comment by fabriciomurta** [comment]
This change will imply the need for the displayed date (in blog posts, for instance), instead of statically printed out, to be dynamically calculated to use the browser's local time zone (and not just be output by retype), if you want pinpointed publish times and nobody complaining about "it was published 10 hours late!" from their +10h time offset. :)

**Reply by RickCogley** [reply]
Sure, there's a few aspects to consider. If it's used for scheduling posts it should be exact, in my opinion. One wrinkle to consider wrt browser time is, people use VPNs these days.
